asciidoc/courses/llm-fundamentals/modules/1-introduction/lessons/1-neo4j-and-genai/questions/2-fixing-hallucination.adoc

```diff
@@ -0,0 +1,22 @@
+[.question]
+= 2. LLM Hallucinations
+
+Why might an LLM produce outputs that manifest as generating untrue facts or nonsensical explanations?
+
+* [ ] LLMs are not designed to generate human-like text.
+* [*] LLMs rely on patterns and sometimes overfit to the data they've been trained on.
```

Comment:
I think you can work out this answer through the process of illimintation but word "overfit" doesnt appear in the lesson and it might take a leap to connect here.